tool/src/org/antlr/v4/codegen/target/GoTarget.java

```diff
@@ -20,12 +20,20 @@
 public class GoTarget extends Target {
 
 	protected static final String[] goKeywords = {
-        "break","default","func","interface","select",
-        "case","defer","go","map","struct",
-        "chan","else","goto","package","switch",
-        "const","fallthrough","if","range","type",
-        "continue","for","import","return","var"
-    };
+			"break", "default", "func", "interface", "select",
+			"case", "defer", "go", "map", "struct",
+			"chan", "else", "goto", "package", "switch",
+			"const", "fallthrough", "if", "range", "type",
+			"continue", "for", "import", "return", "var",
+
+			// predeclared identifiers https://golang.org/ref/spec#Predeclared_identifiers
```

Comment:
What do you think about putting predeclared identifiers into a separate predeclared identifier array and then combining both arrays together where they're needed? It seems misleading to put predeclared identifiers into a keyword array. Or just rename goKeywords to something like goReservedWords?